src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,15 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="com.bugsnag.android"
   android:versionCode="1"
-  android:versionName="1.0" >
+  android:versionName="1.0">
+
+  <!-- Required: Used to deliver Bugsnag crash reports -->
+  <uses-permission android:name="android.permission.INTERNET" />
+
+  <!-- Optional: To provide network connectivity information to Bugsnag -->
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+
+  <!-- Optional: To see which Activity was active at the time of a crash -->
+  <uses-permission android:name="android.permission.GET_TASKS" />
```

Comment:
I wouldn't put these two here as they are automatically added to all consumers requiring opt-out. Opt-in seems a much better strategy for the optional permissions.